instrumentation/net/http/otelhttp/transport.go

```diff
@@ -65,7 +66,7 @@ func (t *Transport) applyConfig(c *config) {
 }
 
 func defaultTransportFormatter(_ string, r *http.Request) string {
-	return r.Method
+	return fmt.Sprintf("HTTP %s", r.Method)
```

Comment:
```suggestion
	return "HTTP " + r.Method
```

Can probably use simple string concatenation here.